docs/admin/kubeadm.md

```diff
@@ -820,6 +820,58 @@ This means you can, for example, prepopulate `/etc/kubernetes/pki/ca.crt`
 and `/etc/kubernetes/pki/ca.key` with an existing CA, which then will be used
 for signing the rest of the certs.
 
+## Self-hosting the Kubernetes control plane  {#self-hosting}
+As of 1.8, kubeadm can experimentally create a _self-hosted_ Kubernetes control
+plane. This means that key components such as the API server, controller
+manager, and scheduler run as [DaemonSet pods](/docs/concepts/workloads/controllers/daemonset/)
+configured via the Kubernetes API instead of [static pods](/docs/tasks/administer-cluster/static-pod/)
+configured in the kubelet via static files.
+
+Self-hosting is alpha in kubeadm 1.8 but is expected to become the default in
+a future version. To create a self-hosted cluster, pass the `--feature-gates=SelfHosting=true`
+flag to `kubeadm init`.
+
+#### Caveats
+Kubeadm self-hosting in 1.8 has some important limitations. In particular, a
+self-hosted cluster cannot currently recover from a simultaneous reboot of all
```

Comment:
of any master node. No matter how many of them you have, the one you reboot won't come back ;)
Also we shouldn't imply that we support more than one master node as other caveats take place then

Reply:
Ah, I see. I guess because the master node's kubelet is configured to talk to the localhost host port so it won't even come back online if you did manually set up multiple masters. I'll clarify.

Reply:
Do we have any ideas for a manual procedure to recover a cluster in this state?